applications/MeshingApplication/custom_strategies/custom_convergencecriterias/error_mesh_criteria.h

```diff
@@ -297,16 +297,18 @@ class ErrorMeshCriteria : public virtual  ConvergenceCriteria< TSparseSpace, TDe
             // Remeshing
             if (mRemeshingUtilities == MMG)
             {
-                if (mDimension == 2)
-                {
-                    MmgProcess<2> MmgRemesh = MmgProcess<2>(mThisModelPart, mThisParameters["remeshing_parameters"]); 
-                    MmgRemesh.Execute();
-                }
-                else
-                {
-                    MmgProcess<3> MmgRemesh = MmgProcess<3>(mThisModelPart, mThisParameters["remeshing_parameters"]); 
-                    MmgRemesh.Execute();
-                }
+                #ifdef INCLUDE_MMG
+                    if (mDimension == 2)
+                    {
+                        MmgProcess<2> MmgRemesh = MmgProcess<2>(mThisModelPart, mThisParameters["remeshing_parameters"]); 
+                        MmgRemesh.Execute();
+                    }
+                    else
+                    {
+                        MmgProcess<3> MmgRemesh = MmgProcess<3>(mThisModelPart, mThisParameters["remeshing_parameters"]); 
+                        MmgRemesh.Execute();
+                    }
+                #endif
```

Comment:
Would it make sense to throw an error if you get here but don't have MMG compiled? If I understand this correctly, this will just silently do nothing if one does not have MMG